Source/svg/SVGParser.swift

```diff
@@ -1697,3 +1706,12 @@ fileprivate extension String {
         return String(self[start..<end])
     }
 }
+
+fileprivate class UserSpacePath: Path {
+    let userSpace: Bool
+
+    init(segments: [PathSegment], userSpace: Bool) {
+        self.userSpace = userSpace
+        super.init(segments: segments, fillRule: .nonzero)
```

Comment:
Looks like you just ignore `fillRule` here, that's why composing better than inheritance :-)